torch/linalg/__init__.py

```diff
@@ -204,31 +211,36 @@
 slogdet = _add_docstr(_linalg.linalg_slogdet, r"""
 linalg.slogdet(input, *, out=None) -> (Tensor, Tensor)
 
-Calculates the sign and natural logarithm of the absolute value of a square matrix's determinant,
-or of the absolute values of the determinants of a batch of square matrices :attr:`input`.
-The determinant can be computed with ``sign * exp(logabsdet)``.
+Computes the sign and logarithm of the norm of the determinant of a square matrix. Supports batched inputs.
+
+For a complex :attr:`input`, it returns the angle and the logarithm of the modulus of the
```

Comment:
We can settle for one. I do not have any strong preference. What do you call |z| for a complex number? I like norm, but I'm happy with any of the options. I like "absolute value" a bit less, as it tends to be used a lot for real numbers.

Reply:
I changed it to "modulus" everywhere, is that fine?

Reply:
Yes, I think it's fine since we are talking about math. Note that in programming it may mean a different thing: https://en.wiktionary.org/wiki/modulus#Noun

Reply:
Agree, but when you talk about the modulus in the other sense, it's always "the modulus with respect to  `p`" (or other number), so I think the confusion would be minimum. But thanks for bringing that up!